aviary/variable_info/variable_meta_data.py

```diff
@@ -6028,6 +6092,17 @@
          'vehicle, used as an additional degree of control for hybrid engines'
 )
 
+add_meta_data(
+    Dynamic.Mission.INSTALLATION_LOSS_FACTOR,
+    meta_data=_MetaData,
+    historical_name={"GASP": 'INGASP.FT',
```

Comment:
This isn't set in the input deck, so it should not have `INGASP`
```suggestion
    historical_name={"GASP": 'FT',
```